packages/starboard-python/src/run.ts

```diff
@@ -69,6 +69,12 @@ export async function runStarboardPython(
             div.className = "rendered_html cell-output-html";
             div.appendChild(new DOMParser().parseFromString(result, "text/html").body.firstChild as any);
             htmlOutput.appendChild(div);
+            // Just putting HTML with script tags on the DOM will not get them evaluated
+            // Using this hack we execute them anyway
+            // TODO Don't re-evaluate all cells, but only do this for the current cell
+            document.querySelectorAll('div.cell-bottom * script[type|="text/javascript"]').forEach(
```

Comment:
```suggestion
            div.querySelectorAll('script[type|="text/javascript"]').forEach(
```
might resolve the TODO